jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -110,6 +110,15 @@
           ></v-select>
         </v-row>
 
+        <v-row v-if="current_plot_type != 'Curve of Growth'">

```

Comment:
Everywhere else specifically mentions radial profile, so maybe this would be more future-proof (although less-readable) as:

```suggestion
        <v-row v-if="current_plot_type.indexOf('Radial Profile') != -1">

```

Reply:
Doesn't hurt. Thanks!